src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1108,12 +1116,25 @@ protected async Task<TResult> InvokeCoreAsync<TResult>(long? id, string targetNa
                 {
                     if (!request.IsResponseExpected)
                     {
+                        if (JsonRpcEventSource.Instance.IsEnabled())
+                        {
+                            JsonRpcEventSource.Instance.SendNotificationStart($"Request Method: \"{request.Method}\", Arguments: \"{JsonRpcEventSource.GetArgumentsString(request.Arguments)}\"");
```

Comment:
I'd like to drop arguments from these events. Then we can pass just the `request.Method` string itself as the argument and the event can be raised with `method` as a proper column in the ETW event that can be easily filtered for.

Reply:
Required to identify the endpoint.